.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
i think ideally launch.json should not be in gitignore because if someone adds a new configuration then all the team members should be aware of the config & fields required. Having said that we can mask the value as it may contain some credentials or keys which should be hidden. In this case it is not required.